pkg/datasources/databases.go

```diff
@@ -261,7 +180,7 @@ func ReadDatabases(ctx context.Context, d *schema.ResourceData, meta any) diag.D
 			"retention_time":  database.RetentionTime,
 			"resource_group":  database.ResourceGroup,
 			"owner_role_type": database.OwnerRoleType,
-			"description":     databaseDescription,
+			"describe_output": databaseDescription,
```

Comment:
This should also be extracted (like with warehouse; I wrote about this in the comment on the previous PR: https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2842#discussion_r1637873291)

Reply:
Extracted schemas and converting functions